components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -13,26 +17,25 @@ import CloudAnnouncementBar from './cloud_announcement_bar';
 import PaymentAnnouncementBar from './payment_announcement_bar';
 import CloudTrialAnnouncementBar from './cloud_trial_announcement_bar';
 
-export default class AnnouncementBarController extends React.PureComponent {
-    static propTypes = {
-        license: PropTypes.object,
-        config: PropTypes.object,
-        user: PropTypes.shape({
-            email: PropTypes.string.isRequired,
-            email_verified: PropTypes.bool,
-        }),
-        canViewSystemErrors: PropTypes.bool.isRequired,
-        latestError: PropTypes.object,
-        totalUsers: PropTypes.number,
-        warnMetricsStatus: PropTypes.object,
-        actions: PropTypes.shape({
-            dismissError: PropTypes.func.isRequired,
-        }).isRequired,
-    }
+type Props = {
+    license?: ClientLicense;
+    config?: Partial<ClientConfig>;
+    user?: Partial<UserProfile>;
+    canViewSystemErrors: boolean;
+    latestError?: {
+        error: any;
+    };
+    totalUsers?: number | AnalyticsRow[];
```

Comment:
I don't know what's going on with the type here, but since it isn't used by ConfigurationAnnouncementBar, can we also remove it from here?

Reply:
done